### PR TITLE
Assembling

### PR DESCRIPTION
**Pull request by Azahires**

### CODE REVIEW

**Review by SOA-P — DISMISSED** [review]

File: frontend/src/assets/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+package-lock.json
```
Comment (LoicBrassart) [inline frontend/src/assets/.gitignore]: Meh! o_O

File: frontend/src/components/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+package-lock.json
```
Comment (LoicBrassart) [inline frontend/src/components/.gitignore]: Re-Meh!

File: frontend/src/pages/.gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+.env
```
Comment (LoicBrassart) [inline frontend/src/pages/.gitignore:1]: Noooooooooooooooooooooooooooooooooope

File: package.json [hunk 4]
```diff
@@ -26,5 +26,8 @@
   "bugs": {
     "url": "https://github.com/WildCodeSchool/2022-09-JS-Lille-project-3-g2-externatic/issues"
   },
-  "homepage": "https://github.com/WildCodeSchool/2022-09-JS-Lille-project-3-g2-externatic#readme"
+  "homepage": "https://github.com/WildCodeSchool/2022-09-JS-Lille-project-3-g2-externatic#readme",
+  "dependencies": {
+    "prop-types": "^15.8.1"
+  }
```
Comment (LoicBrassart) [inline package.json]: Should be in the `frontend` folder

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Fix minor Evils

**Review by LoicBrassart — APPROVED** [review]
SnE

**Review by SOA-P — APPROVED** [review]

**Pull request merged**